### DISCUSSION TITLE
How many years are celebrated on a ruby anniversary?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#534] [Galaxy Trivia] [answered]
**Bonus:** What is the name of the anniversary after ruby?

**Comment by peterbaccaro** [comment]
The **40th** Wedding Anniversary is known as the Ruby Anniversary.
The **Sapphire** Anniversary which is the 45th Wedding Anniversary is the next one after Ruby.

**Comment by AlienegraGeek** [comment, marked as answer]
A ruby anniversary celebrates 40 years of marriage. It's a significant milestone for couples, and the ruby gemstone symbolizes the love and passion that has endured throughout their years together.

The anniversary that comes after the ruby anniversary is the 45th anniversary, which is traditionally known as the Sapphire Anniversary.
